recipe/conda_build_config.yaml

```diff
@@ -390,6 +390,7 @@ python:
   - 2.7
   - 3.5
   - 3.6
+  - 3.7
```

Comment:
You have to change the compilers for Windows too. Let's discuss this a bit before merging.

Reply:
Not sure what you mean? Are the new Python 3.7 package built with another compiler?

Reply:
BTW, we are sprinting here are SciPy and there are tons of people waiting on these features that are being blocked by "let's wait...

Reply:
> Not sure what you mean? Are the new Python 3.7 package built with another compiler?

No, it's still vs2015, but you have to tell conda-build that.

Reply:
> BTW, we are sprinting here are SciPy and there are tons of people waiting on these features that are being blocked by "let's wait..

My concern was that this would suddenly start creating python 3.7 builds for every package and it would pull dependencies from defaults leading to incompatibilities. I like the idea of the bot making PRs.

I don't know what you mean by being blocked. If you want 3.7 builds, it's as simple as adding a `conda_build_config.yaml` to the recipe folder and rerendering.

Reply:
Yes, the new compiler migration is blocked on a "forever wait" plead. We need to get the ball rolling there.

Reply:
We could stage this (and probably the dual compiler) somewhere special and have the bot do both of these in graph order together?